experimental/kubernetes/spinnaker-local.yml

```diff
@@ -7,12 +7,7 @@ providers:
     # If you want to deploy some services to Amazon Web Services (AWS),
     # set enabled and provide primary credentials for deploying.
     # Enabling AWS is independent of other providers.
-    enabled: true
-    defaultRegion: us-east-1
-    defaultIAMRole: BaseIAMRole
-    primaryCredentials:
-      name: my-aws-account
-      # Store actual credentials in /home/spinnaker/.aws/credentials
+    enabled: false
```

Comment:
Would it make sense to leave the rest of the aws config in place, and just set `enabled` to `false`? Otherwise one would have to go hunting around for the other attributes that need to be configured when flipping that flag.